### ISSUE TITLE
Feature Request: Quick Output Device Switching

### ISSUE THREAD

**Issue by sharpau** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#3012]
OS: Windows 10

I use GPMDP while I'm working. I tend to switch several times a day between music on speakers, music on headphones, and often want the music on one device while my OS audio output is elsewhere. While this all works OK, it's just a lot of clicks to get to the desktop settings and change the music output. It would make me incredibly happy to have an audio output switcher easily available - for instance, if there were a little dropdown next to the Chromecast button when music is playing.

In other words, right now it takes me 6 clicks to change the audio output - sidebar, desktop settings, playback, open the dropdown, select output, close settings modal. In my dreams it's two clicks - drop the dropdown and click the device I want.

Thanks for the project!

**Comment by MarshallOfSound** [comment]
This already exists, you can change the audio output device from the tray menu 👍

**Comment by sharpau** [comment]
!!! THANK YOU
